lms/djangoapps/email_marketing/models.py

```diff
@@ -0,0 +1,57 @@
+"""
+Email-marketing-related models.
+"""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from config_models.models import ConfigurationModel
+
+
+class EmailMarketingConfiguration(ConfigurationModel):
+    """ Email marketing configuration """
+
+    class Meta(object):
+        app_label = "email_marketing"
+
+    sailthru_key = models.fields.CharField(
+        max_length=32,
+        help_text=_(
+            "API key for accessing Sailthru. "
+        )
+    )
+
+    sailthru_secret = models.fields.CharField(
+        max_length=32,
+        help_text=_(
+            "API secret for accessing Sailthru. "
+        )
+    )
+
+    sailthru_new_user_list = models.fields.CharField(
+        max_length=32,
```

Comment:
They don't document a restriction.  It's just a list name which will be set to 'All edX Users'